invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -100,7 +125,7 @@ computer_file_characteristics:
 
 # 260__b 260__c
 
-imprint:
+publication:
     creator:
```

Comment:
It would be good to avoid changes such as imprint -> publication unless absolutely necessary, because we have to go and change format templates to use the new name... Let's rather stick to old names and we can revise names when we shall be migrating to the fully new data model. #999